src/Goerli-DssSpell.sol

```diff
@@ -56,108 +50,20 @@ contract DssSpellAction is DssAction {
-            //       changes to an OSM.
-            DssExecLib.authorize(_clip, DssExecLib.clipperMom());
-            DssExecLib.setValue(_clip, "stopped", 0);
-            // Use Abacus/LinearDecrease
-            DssExecLib.setContract(_clip, "calc", MCD_CLIP_CALC_PAXUSD_A);
-            // Set Liquidation Penalty to 0
-            DssExecLib.setIlkLiquidationPenalty(_ilk, 0);
-            // Set Auction Price Multiplier (buf) to 1
-            DssExecLib.setStartingPriceMultiplicativeFactor(_ilk, 100_00);
-            // Set Local Liquidation Limit (ilk.hole) to 3m DAI
-            DssExecLib.setIlkMaxLiquidationAmount(_ilk, 3_000_000);
-            // Set tau for Abacus/LinearDecrease to 4,320,000 second
-            DssExecLib.setLinearDecrease(MCD_CLIP_CALC_PAXUSD_A, 4_320_000);
-            // Set Max Auction Duration (tail) to 43,200 seconds
-            DssExecLib.setAuctionTimeBeforeReset(_ilk, 43_200);
-            // Set Max Auction Drawdown / Permitted Drop (cusp) to 0.99
-            DssExecLib.setAuctionPermittedDrop(_ilk, 99_00);
-            // Set Proportional Kick Incentive (chip) to 0
-            DssExecLib.setKeeperIncentivePercent(_ilk, 0);
-            // Set Flat Kick Incentive (tip) to 0
-            DssExecLib.setKeeperIncentiveFlatRate(_ilk, 0);
-        }
 
         // ------------------ Setup new Starknet Governance Relay -----------------
         // Forum: https://forum.makerdao.com/t/starknet-changes-for-executive-spell-on-the-week-of-2022-11-29/18818
         // Relay l2 part of the spell
-        // L2 Spell: https://goerli.voyager.online/contract/0x04c93f9818a4f81f6f2c6f0f660cb4986b789b6b6fb1b274b879649deed74eb8#code
-        StarknetGovRelayLike(STARKNET_GOV_RELAY).relay(L2_GOV_RELAY_SPELL);
+        // L2 Spell:
+        // https://goerli.voyager.online/contract/0x05a958b692c498791a215f0624a105393f5b33cfdc3fb51bdf17e629b766119a#code
+        // it is a fix of 2022-11-28 l2 spell:
+        // https://goerli.voyager.online/contract/0x04c93f9818a4f81f6f2c6f0f660cb4986b789b6b6fb1b274b879649deed74eb8#code
+        // that did not executed correctly due to the incorrect bridge_legacy address:
+        // https://goerli.voyager.online/tx/0x6057a9ac5d03e09da99c9f07f725af5170eec6d4364f3a7f8dbc03362706698
 
-        // Configure Chainlog
-        DssExecLib.setChangelogAddress("STARKNET_GOV_RELAY_LEGACY", STARKNET_GOV_RELAY);
-        DssExecLib.setChangelogAddress("STARKNET_GOV_RELAY", NEW_STARKNET_GOV_RELAY);
-        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_GUSD_A", MCD_CLIP_CALC_GUSD_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_USDC_A", MCD_CLIP_CALC_USDC_A);
-        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_PAXUSD_A", MCD_CLIP_CALC_PAXUSD_A);
+        StarknetGovRelayLike(STARKNET_GOV_RELAY).relay(L2_GOV_RELAY_SPELL);
 
-        DssExecLib.setChangelogVersion("1.14.6");
+        DssExecLib.setChangelogVersion("1.14.7");
```

Comment:
If there are no changes in the chainlog, the bump can be removed.

Reply:
Even if there was a change in the chainlog we shouldn't have made the bump as this was a fix hitting goerli before the same change happened on mainnet.